# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1458939481 14400
#      Fri Mar 25 16:58:01 2016 -0400
# Branch stable
# Node ID e09f4c9f800ab8dca506ec34edde20fe7407f442
# Parent  f744aa689d94cac45138bc99adf3d0b1fc71ba9e
macros.texi: Colorized links for PDF files with Texinfo 6.x.

diff --git a/doc/interpreter/macros.texi b/doc/interpreter/macros.texi
--- a/doc/interpreter/macros.texi
+++ b/doc/interpreter/macros.texi
@@ -81,8 +81,29 @@ See also: \args\.
 
 @c FIXME: someday, when Texinfo 5.X is standard, we might replace this with
 @c @backslashchar, which is a new addition to Texinfo.
 
 @macro xbackslashchar
 \\
 @end macro
 
+@c These may be useful for all, not just for octave.texi.
+@tex
+  \ifx\rgbDarkRed\thisisundefined
+    \def\rgbDarkRed{0.50 0.09 0.12}
+  \fi
+  \ifx\linkcolor\thisisundefined
+    \relax
+  \else
+    \global\def\linkcolor{\rgbDarkRed}
+  \fi
+  \ifx\urlcolor\thisisundefined
+    \relax
+  \else
+    \global\def\urlcolor{\rgbDarkRed}
+  \fi
+  \ifx\urefurlonlylinktrue\thisisundefined
+    \relax
+  \else
+    \global\urefurlonlylinktrue
+  \fi
+@end tex
